ui/Button/Button.module.scss

```diff
@@ -0,0 +1,127 @@
+@import "variables";
+@import "mixins";
+
+.button {
+    @include border-radius-m;
+    @include text-label;
+
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    padding: $spacing-2 $spacing-3;
+    background: none;
+    font-weight: $font-weight-bold;
+    text-decoration: none;
+    appearance: none;
+    box-shadow: none;
+    cursor: pointer;
+    border: 1px solid transparent;
+    transition: background-color 0.15s ease-in, border-color 0.15s ease-in;
+
+    &:disabled {
+        cursor: not-allowed;
+        opacity: 0.4;
+    }
+
+    &.loading {
+        cursor: wait;
+    }
+}
+
+.primary {
+    background-color: var(--prezly-accent-color);
+    border-color: var(--prezly-accent-color);
+    color: $color-base-white;
```

Comment:
## This branch

![image](https://user-images.githubusercontent.com/67554982/226354439-458c643b-e830-4d75-aa96-660d2cc82e25.png)

## Prod

![image](https://user-images.githubusercontent.com/67554982/226354566-c469b388-6847-4a62-b0d4-41473370261a.png)


Should we treat this as a regression or an improvment? Honestly, it looks better in this branch, so I guess it's fine :)